view/adminhtml/templates/info/adyen_pos_cloud.phtml

```diff
@@ -61,3 +61,10 @@ $_isDemoMode = $block->isDemoMode();
 <?php endif;?>
 
 <?php echo $block->getChildHtml()?>
+    <style>
+        table.terminal-api-receipt{margin-top: 10px}
+        td.terminal-api-receipt-name{padding:inherit; font-weight: 700}
+        td.terminal-api-receipt-value{padding:inherit;}
+        tr.terminal-api-receipt{padding-bottom: 5px;}
+    </style>
+<table><?php $receipt = $this->getMethod()->getInfoInstance()->getAdditionalInformation('receipt'); echo $receipt;?></table>
```

Comment:
do we need `<table>` since there is already one in receipt variable?
Let's also use multiple lines